velox/exec/TaskStructs.h

```diff
@@ -43,7 +43,7 @@ struct BarrierState {
   /// all hashtables from the peers and assembles them into one (HashBuilder
   /// operator does that). After the last drier done its work, the promises are
   /// fulfilled and the non-last drivers can continue.
-  std::vector<ContinuePromise> promises;
```

Comment:
I don't think we need this rename as the structure is designed for that purpose.

Reply:
@xiaoxmeng 
Thanks for taking time reviewing!
I believe we need this rename - it make code browsing much easier, especially for people unfamiliar with the code.